misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -382,15 +422,48 @@ private Document getDocumentFromModule(Module moduleInstance, String sourceFileN
         return document;
     }
 
-    private List<Path> getExclusionJarList(JBallerinaBackend jBallerinaBackend) {
+    private List<Path> getExclusionJarList(JBallerinaBackend jBallerinaBackend, Package pkg) {
         List<Path> exclusionPathList = new ArrayList<>(getDependencyJarList(jBallerinaBackend));
         exclusionPathList.add(jBallerinaBackend.runtimeLibrary().path());
         for (JarLibrary library : ProjectUtils.testDependencies()) {
             exclusionPathList.add(library.path());
         }
+        // Add platform libs of this package to exclusion jar list
+        exclusionPathList.addAll(getPlatformLibsList(jBallerinaBackend, pkg));
         return exclusionPathList;
     }
 
+    private List<Path> getCodeGeneratedJarList(JBallerinaBackend jBallerinaBackend) {
```

Comment:
Looks like there is no usage for this method.

Reply:
Fixed with 1fabfc7e25ef44de12795d3e47d8a39b0d2efede